From 5ef207b5f0fbafe4172c397faa5512ed51599cbd Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Tue, 30 Jul 2024 15:32:44 +0200
Subject: [PATCH 661/841] feat(stm32mp2-fdts): fix rcc support in stm32mp251

Alignment with linux kernel device tree, use the correct binding.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 16c70230bf631daaaacb5514fe845723bdb997ff feat(stm32mp2-fdts): fix rcc support in stm32mp251]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: Ib3dc18dac2b3b4b8111cbd57021298ec8a250ee1
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/396007
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gabriel FERNANDEZ <gabriel.fernandez@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
---
 fdts/stm32mp251.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fdts/stm32mp251.dtsi b/fdts/stm32mp251.dtsi
index c3f93861a..378f11f96 100644
--- a/fdts/stm32mp251.dtsi
+++ b/fdts/stm32mp251.dtsi
@@ -396,7 +396,7 @@
 			};
 		};
 
-		rcc: rcc@44200000 {
+		rcc: clock-controller@44200000 {
 			compatible = "st,stm32mp25-rcc";
 			reg = <0x44200000 0x10000>;
 			#clock-cells = <1>;
-- 
2.47.2

